src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -66,14 +71,14 @@ public void parse_allFieldsPresent_success() {
 
         // whitespace only preamble
         assertParseSuccess(parser, PREAMBLE_WHITESPACE + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
-                + NRIC_DESC_BOB + DOB_DESC_BOB + GENDER_DESC_BOB + ADDRESS_DESC_BOB,
-                new AddCommand(expectedPerson));
+                + NRIC_DESC_BOB + DOB_DESC_BOB + GENDER_DESC_BOB + ADDRESS_DESC_BOB + PRIORITY_DESC_BOB,
+                           new AddCommand(expectedPerson));
```

Comment:
I believe there is no need for `PRIORITY_DESC_BOB` here since `add` will only require `NRIC`, `GENDER`, `ADDRESS`, `DATEOFBIRTH`, `NAME` and `PHONE`